src/TLuaInterpreter.cpp

```diff
@@ -13917,8 +15162,17 @@ void TLuaInterpreter::initLuaGlobals()
     lua_register(pGlobalLua, "tempColorTrigger", TLuaInterpreter::tempColorTrigger);
     lua_register(pGlobalLua, "isAnsiFgColor", TLuaInterpreter::isAnsiFgColor);
     lua_register(pGlobalLua, "isAnsiBgColor", TLuaInterpreter::isAnsiBgColor);
-    lua_register(pGlobalLua, "stopSounds", TLuaInterpreter::stopSounds);
+    lua_register(pGlobalLua, "receiveMSP", TLuaInterpreter::receiveMSP);
+    lua_register(pGlobalLua, "loadSoundFile", TLuaInterpreter::loadSoundFile);
+    lua_register(pGlobalLua, "loadMusicFile", TLuaInterpreter::loadMusicFile);
     lua_register(pGlobalLua, "playSoundFile", TLuaInterpreter::playSoundFile);
+    lua_register(pGlobalLua, "playMusicFile", TLuaInterpreter::playMusicFile);
+    lua_register(pGlobalLua, "stopAllMedia", TLuaInterpreter::stopAllMedia);
+    lua_register(pGlobalLua, "stopAllMusic", TLuaInterpreter::stopAllMusic);
+    lua_register(pGlobalLua, "stopAllSounds", TLuaInterpreter::stopAllSounds);
```

Comment:
`stopSounds()` does this already - maybe we don't need it `stopAllSounds()` as well?